awx/main/models/label.py

```diff
@@ -36,14 +38,31 @@ def get_absolute_url(self, request=None):
 
     @staticmethod
     def get_orphaned_labels():
-        return Label.objects.filter(organization=None, unifiedjobtemplate_labels__isnull=True, inventory_labels__isnull=True)
+        return Label.objects.filter(
+            organization=None,
+            unifiedjobtemplate_labels__isnull=True,
+            inventory_labels__isnull=True,
+            schedule_labels__isnull=True,
+            workflowjobtemplatenode_labels__isnull=True,
+            workflowjobnode_labels=True,
+        )
 
     def is_detached(self):
-        return Label.objects.filter(id=self.id, unifiedjob_labels__isnull=True, unifiedjobtemplate_labels__isnull=True, inventory_labels__isnull=True).exists()
+        return Label.objects.filter(
+            id=self.id,
+            unifiedjob_labels__isnull=True,
```

Comment:
All these kwargs are repeated in `get_orphaned_labels` except for this one. Is there are reason for that?

Reply:
I'm not sure why they left that out initially but as you pointed out, this method is never referenced anywhere so I chopped it out.